### DISCUSSION TITLE
Use github discussions?

### DISCUSSION THREAD

**Discussion by stefanbuehler** [atmtools/arts#806] [Polls]
Some people say that github discussions are the thing now.

**Comment by erikssonpatrick** [comment]
For what purpose? Development, user support or ... Would it replace something that we already have?
(Not familiar with github discussions, so sorry if the answer is obvious)

**Reply by riclarsson** [reply]
GitHub discussions is a forum that can link between pull requests (e.g., PRs: #805, or Issues #215) and tag people (e.g., @erikssonpatrick).  You can see how it is used in other projects that has enabled it.  You can reach the forum from the main repository [site](https://github.com/atmtools/arts).  It's in the top toolbar, between Pull Request and Actions. We have just activated it to hold this poll.

This would work as a replacement for the user and development mailing lists.  It has the advantage over multiple mailing lists that it is directly connected to our development.  It has the dis-/advantage that we no longer would be using the mailing list - there are some 30 people signed up to the mailing list.  The advantage of this is less maintenance, and the disadvantage is that we can not longer know who is signed up to receive information, just the number of people interested in our project (by the Watch/Unwatch counter on the main page).  The maintenance bit is important.  If we can get this going and Oliver can stop filtering mails going to spam, that would be quite helpful.

In my opinion, it is simply better than a mailing list because we will be able to easily reference other discussion threads, PRs, and Issues.  It consolidates all discussions around ARTS to our GitHub page, making the user-generated information more readily findable.  Another thing I am hopeful about is that tools like GitHub co-pilot might pick up on our discussions here more easily to offer better suggestions to auto-complete repeated tasks (from it's answers about ARTS, it might be reading the archive).

The issue with the email list is that very few people are engaged in answering questions and that answers are lost to time because the email list search is not graphically appealing (compare the [archive](https://www.mail-archive.com/arts_users.mi@lists.uni-hamburg.de/) to what [the discussion page](https://github.com/atmtools/arts/discussions) looks like - better if you find a project that has had discussion active).

We would still like to maintain an emailing list for pure announcements and workshop calls.  Our Hamburg discussion on this indicated we want to consolidate this in just a single ARTS mailing list where only we are allowed to send information.

The old mailing lists will remain active for some time in a transition mode.  This transition mode would be that only @olemke receives the email and simply forward the user/developer to our GitHub forum to restate their question.
